community/blog/2020-03-Releasing-Arcus-Messaging-and-BackgroundJobs-v.0.1.0/README.md

```diff
@@ -0,0 +1,79 @@
+Both help in in writing repeating jobs that securely receive external events and process them any way you want.
+
+Come and take a look at what they have to offer!
+
+## Receiving Messages From Azure Service Bus Queues & Topics
+
+With Arcus Messaging we provide something that's called a **message pump** for messages on Azure Service Bus.
+This pump can be registered very easily on startup of your application.
+
+Once registered, one or more **message handlers** can be registered that will eventually process the message. 
+Based on the message type, the pump will send the message to the correct registered handler.
+This allows a loosely coupled system between receiving and processing.
+
+By using our [`Arcus.Messaging.Pumps.ServiceBus`](https://www.nuget.org/packages/Arcus.Messaging.Pumps.ServiceBus/) NuGet package, you can configure such a pump during startup.
+See the following section as an example to handle `Order` messages with a custom `OrdersMessageHandler` implementation. 
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddServiceBusQueueMessagePump("ServiceBus:Queue:ConnectionString")
+            .WithServiceBusMessageHandler<OrdersMessageHandler, Order>();
+}
+```
+
+Where the your `OrderMessageHandler` will process the `Order` messages received on the Azure Service Bus Queue:
+
+```csharp
+public class OrdersMessageHandler : IMessageHandler<Order>
+{
+    public async Task ProcessMessageAsync(
+        Order message, 
+        MessageContext messageContext, 
+        MessageCorrelationInfo correlationInfo, 
+        CancellationToken cancellationToken)
+    {
+    }
+}
+```
+
+See the [docs](https://messaging.arcus-azure.net/features/message-pumps/service-bus) for more info.
+
+## Monitoring Health of a Worker
+
+With Arcus Messaging we provide provide the capability to expose a TCP health probe endpoint that allows processes to periodically check the liveness/readiness of the application which is extremely important in container workloads.
+
+This allows you to expose a health endpoint without having to include the HTTP stack in your workers, keeping your application simple.
+Instead of reinventing the wheel, we've decided to build upon the built-in [.NET Core Health Checks](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/health-checks?view=aspnetcore-3.1) which allows us to integrate with existing tooling and leveraging a very compatible system.
+
+By using our [`Arcus.Messaging.Health`](https://www.nuget.org/packages/Arcus.Messaging.Health/) NuGet package, you can very easily opt-in to expose the TCP endpoint:
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddTcpHealthProbes("Health:Tcp:Port");
```

Comment:
Where did our example of health checks go? now it's not clear that it uses this in the sample

Reply:
Well, it already registered the health checks services when calling `.AddTcpHealthProbes`, and the other one about blob storage I removed bc it was a bit clutter, no?

Reply:
It's more about showing how to use it and afaik you can still use the scenario that was mentioned and use the overload as an alternative but it's not a must.

If the sample that was provided is used people will more easily see the value because they know that part; now it looks like magic/no value because no health checks are configured

Reply:
Ok! I changed it to the sample you proposed.